webpack.config.prod.js

```diff
@@ -0,0 +1,55 @@
+const path = require('path');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  mode: 'production',
+  entry: './amt/site/static/ts/amt.ts',
+  output: {
+    filename: 'amt.[contenthash].js',
+    path: path.resolve(__dirname, 'amt/site/static/dist'),
+    publicPath: 'amt/site/static/dist'
```

Comment:
This path needs a trailing / for paths to resolve correctly:

```suggestion
    publicPath: 'amt/site/static/dist/'
```